backend/src/v5/schemas/tickets/templates.js

```diff
@@ -143,7 +149,7 @@ const propSchema = Yup.object().shape({
 
 		if (type === propTypes.ANY_OF) return res.oneOf(values);
 
-		if (type === propTypes.IMAGE || type === propTypes.IMAGE_LIST) return Yup.mixed().strip();
+		if (complexTypes.has(type)) return Yup.mixed();
```

Comment:
I would put somehting like this here

```
if (complexTypes.includes(type)) {
			return Yup.mixed().test('invalid-default', `Default property cannot be applied to ${type}`, (value) => !value);
		}
```

havent tested it fully but I think this would do the trick and then you won't need the noDefaultsOnComplexTypesTest test case